modules/service/src/main/scala/lucuma/odb/graphql/mapping/ProposalTypeMapping.scala

```diff
@@ -35,7 +35,7 @@ trait ProposalTypeMapping[F[_]] extends BaseMapping[F]
       discriminator = proposalTypeDiscriminator,
       fieldMappings = List(
         SqlField("id", ProposalView.ProgramId, key = true, hidden = true),
-        SqlField("scienceSubtype", ProposalView.ScienceSubtype, discriminator = true),
+        SqlField("scienceSubtype", ProposalView.ScienceSubtype, discriminator = true, hidden = true),
```

Comment:
Do discriminators need to be marked `hidden` even if they appear in the GraphQL schema?

Reply:
The validator isn't complaining anymore so I think maybe this was a bug that was inadvertently fixed? I'll switch them back where I changed them.

Reply:
:+1: I wonder if it is a bug that the validator _doesn't_ complain about a hidden mapping for something that appears in the schema.

Reply:
Ah good point.